web/vtadmin/src/components/routes/keyspaces/KeyspaceActions.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import Dropdown from '../../dropdown/Dropdown';
+import MenuItem from '../../dropdown/MenuItem';
+import { Icons } from '../../Icon';
+import Toggle from '../../toggle/Toggle';
+import { useValidateKeyspace, useValidateSchemaKeyspace, useValidateVersionKeyspace } from '../../../hooks/api';
+import KeyspaceAction from './KeyspaceAction';
+
+interface KeyspaceActionsProps {
+    keyspace: string;
+    clusterID: string;
+}
+
+const KeyspaceActions: React.FC<KeyspaceActionsProps> = ({ keyspace, clusterID }) => {
+    const [currentDialog, setCurrentDialog] = useState<string | null>(null);
+    const closeDialog = () => setCurrentDialog(null);
+
+    // Validate keyspace
```

Comment:
Tiny nit but FWIW I don't think we need these comments since the behaviour is well described by the function names :)

Reply:
🙏 That's music to my ears haha!